AutoNode/blockchain.py

```diff
@@ -0,0 +1,103 @@
+import json
+
+import requests
+
+from .common import (
+    node_config
+)
+
+_headers = {
+    'Content-Type': 'application/json'
+}
+
+
+def get_current_epoch(endpoint=node_config['endpoint']):
+    return int(get_latest_header(endpoint)["epoch"])
+
+
+def get_latest_header(endpoint=node_config['endpoint']):
+    payload = json.dumps({"id": "1", "jsonrpc": "2.0",
+                          "method": "hmy_latestHeader",
+                          "params": []})
+    response = requests.request('POST', endpoint, headers=_headers, data=payload, allow_redirects=False, timeout=3)
+    body = json.loads(response.content)
+    if 'error' in body:
+        raise RuntimeError(str(body['error']))
+    return body["result"]
+
+
+def get_latest_headers(endpoint=node_config['endpoint']):
+    payload = json.dumps({"id": "1", "jsonrpc": "2.0",
```

Comment:
these are all the same - so why so many same functions? one function with the payload as parameter would capture these like 5 functions

Reply:
Refactored it to 1 underlying function call.